conda/recipes/librmm/meta.yaml

```diff
@@ -21,7 +21,8 @@ requirements:
     - {{ compiler('cuda') }} {{ cuda_version }}
```

Comment:
Don't we need to setup the `conda_build_config.yaml` to select between the `nvcc` and `cuda-nvcc` package here?

Reply:
Nope. This PR is only for backporting cuda-version to our current CUDA 11 builds, which only use nvcc.